cmd/config.go

```diff
@@ -571,6 +589,57 @@ func chooseZone(accountName string, cs *egoscale.Client) (string, error) {
 	return defaultZone, nil
 }
 
+func chooseSSHKey(accountName string, cs *egoscale.Client) (string, error) {
+
```

Comment:
Inconsistent coding style: some functions have a blank first/last line and some don't (personally I prefer without).

Reply:
👍